site2/docs/reference-configuration-websocket.md

```diff
@@ -0,0 +1,495 @@
+# WebSocket
+## Required
+### clusterName
+Name of the cluster to which this broker belongs to
+
+**Default**: `null`
+
+**Dynamic**: `false`
+
+**Category**: 
```

Comment:
If all configurations do not have category values and will not have in the future, consider not showing `category` because it does not provide useful info to users?

Reply:
I think so. However, there're some docs like `reference-configuration-broker` that do have values for `Category`, and docs for `broker` and `websocket` are generated by the same code. We could certainly add some additional conditions to eliminate these useless `Category` and do the trick, but it would made the code hard to extend and clumsy.